tea.go

```diff
@@ -153,7 +154,7 @@ type Program struct {
 
 	// was the altscreen active before releasing the terminal?
 	altScreenWasActive bool
-	ignoreSignals      bool
+	ignoreSignals      atomic.Bool
```

Comment:
`atomic.Bool` was only introduced in go1.19, use `uint32` and `atomic.StoreUint32` instead

Reply:
Indeed!